[JIT] s/uniqueName/debugName

jamesr66a opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#22048 [JIT] s/uniqueName/debugName**

Differential Revision: [D15947873](https://our.internmc.facebook.com/intern/diff/D15947873)

Commit:
[JIT] s/uniqueName/debugName

Krovatkin commented:
yay!!!!!!

apaszke commented:
Sooo what's our plan for having unique names in the future? Just use the node id?

zdevito reviewed (approved):
I think this makes it more clear that uniqueName is a debug name and not stable. It should lead to people using `unique()` or just the pointer itself, which is what we want them to do anyway.

Commit:
Update on "[JIT] s/uniqueName/debugName"

[JIT] s/uniqueName/debugName

gh-metadata: pytorch pytorch 22048 gh/jamesr66a/9/head

Commit:
Update on "[JIT] s/uniqueName/debugName"

[JIT] s/uniqueName/debugName

gh-metadata: pytorch pytorch 22048 gh/jamesr66a/9/head
